test/attr/attr_final.swift

```diff
@@ -1,4 +1,4 @@
-// RUN: %target-typecheck-verify-swift
+// RUN: %target-typecheck-verify-swift -swift-version 4
```

Comment:
Are you referring to the `-swift-version 4` or to the changes to the tests themselves? I thought that all of the source compatibility tests (for v3) were being put int `test/Compatibility`. I added `-swift-version 4` here in an attempt to follow the pattern I saw in `test/attr_[escaping|inlineable|objc]`, but glad to change anything as needed.